test_runner/regress/test_tenant_size.py

```diff
@@ -7,6 +9,34 @@
 from fixtures.types import Lsn
 
 
+def mask_model_inputs(x):
```

Comment:
Could add a comment why this is needed (#2592?) and perhaps an example, or perhaps the example could be a test case, it's a bit difficult to see what does this do.